extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -1048,20 +1061,16 @@ static public function doAfterPost( Status $status, $article, $parentId = 0 ) {
 				wfGetDB( DB_MASTER )->commit();
 				break;
 			default:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 13 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1151)